### DISCUSSION TITLE
Using pylenium with Lamdatest

### DISCUSSION THREAD

**Discussion by nguyet0310** [ElSnoMan/pyleniumio#259] [General]
Here is situation:
- I signup Lamdatest to test username and key:
- I install Pylenium 1.15.2 and Selenium 4.1.5.
- And in pylenium.json, I tried to configure it running with my user name and key from Lamdatest
- I follow instruction in this link to get capabilities for Selenium 4: `https://www.lambdatest.com/capabilities-generator/`
- my pylenium.json is below:
`{
  "driver": {
    "browser": "",
    "remote_url": "https://namekey@hub.lambdatest.com/wd/hub",
    "wait_time": 10,
    "page_load_wait_time": 0,
    "options": [],
    "capabilities": {
      "LT:Options": {
        "user": "my_user_name",
        "accessKey": "my_key",
        "build": "your build name",
        "name": "your test name",
        "platformName": "Windows 11"
      },
      "browserName": "Chrome",
      "browserVersion": "99.0"
    },
    "version": "latest",
    "experimental_options": null,
    "extension_paths": [],
    "webdriver_kwargs": {},
    "seleniumwire_options": {},
    "local_path": ""
  },
  "logging": {
    "screenshots_on": true
  },
  "viewport": {
    "maximize": true,
    "width": 1440,
    "height": 900,
    "orientation": "portrait"
  },

  "customer": {}
}
`
and then run my test with cmd: ` python -m pytest tests/`
but it shows error as below:
`  raise ValueError(f"{browser} is not supported. Cannot build capabilities.")
E           ValueError:  is not supported. Cannot build capabilities.

...x/lib/python3.9/site-packages/pylenium/webdriver_factory.py:58: ValueError`
so my question is how can I make pylenium work with Lamdatest? 
Note: I tried to downgrade to Selenium 3 and then use Selenium 3 capabilities, and it can run Lamdatest, but it shows compatibility issue with Pylenium which require Selenium 4?

**Comment by ElSnoMan** [comment]
Put "chrome" as the value for the driver.browser field in your pylenium.json and try again.

**Reply by nguyet0310** [reply]
it works, thank you. Btw, is it possible to run paralell one for chrome another for FF using Lamdatest ?

**Reply by ElSnoMan** [reply]
Yes, that depends on how you set things up with LambdaTest. They have a matrix option which allows for that although I don't remember exactly how 😅

**Reply by nguyet0310** [reply]
okie, thank you :)
